### DISCUSSION TITLE
Connect to frontend via https?

### DISCUSSION THREAD

**Discussion by abishur** [Koenkk/zigbee2mqtt#12998] [General]
I'm running Home Assistant and Zigbee2MQTT in docker containers.  I've set up home assistant so I can connect to it over https from outside my network and wanted to use an iframe to be able to pull up zigbee2mqtt as well.  When I had it all running on http this work with no issue, but now that it's on https home assistant says it can't load http content when being accessed via https.

How do I configure Zigbee2MQTT so I can connect to the front end using https?

**Comment by Nardol** [comment]
The only solution I see is to put the Zigbee2MQTT behind a reverse proxy which also use HTTPS because the Zigbee2MQTT frontend does not allow to set a certificate to use it with https.
